src/components/TreeReactFlow/TreeReactFlow.stories.tsx

```diff
@@ -139,6 +152,7 @@ export const OddAndEvenMiscStyles: ComponentStory<typeof TreeReactFlow> = (
 ) =>
   treeSized({
     ...args,
+    scrollToDefRef: useRef<ScrollToDef | undefined>(undefined),
```

Comment:
TypeScript complains on this file. I think it would be better to remove all the current changes and just add `scrollToDefRef: { current: undefined }` to the top-level `props` variable.

Reply:
Fixed, thanks for the suggestion. I think we don't compile our Storybook build with TypeScript (for unrelated reasons I can't recall) so this didn't come up in my builds, but anyway Storybook couldn't render the `TreeReactFlow` stories without adding line in each of them, so I'm not sure what happened there.